src/app/pagamentoplanos/service/pagamentoplanos.service.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
import { HttpClient, HttpHeaders } from "@angular/common/http";
```